### PR TITLE
increase brightness for gift card detail page

### PR DESCRIPTION
**Pull request by Serhii-Borodenko**
increase brightness for gift card detail page

### CODE REVIEW

File: lib/src/screens/ionia/cards/ionia_gift_card_detail_page.dart [hunk 1]
```diff
@@ -24,6 +25,12 @@ class IoniaGiftCardDetailPage extends BasePage {
   IoniaGiftCardDetailPage(this.viewModel);
 
   final IoniaGiftCardDetailsViewModel viewModel;
+  double _brightness;
+
+  void increaseBrightness () async {
+    _brightness = await DeviceDisplayBrightness.getBrightness();
+    DeviceDisplayBrightness.setBrightness(1.0);
+  }
```
Comment (mkyq) [inline lib/src/screens/ionia/cards/ionia_gift_card_detail_page.dart:33]: Move this logic into view model, don't have BasePage is Stateless for now, so don't store mutated state there.

**Review by mkyq — CHANGES_REQUESTED** [review]

File: lib/src/screens/ionia/cards/ionia_gift_card_detail_page.dart (hunk 1 above)
Comment (Serhii-Borodenko) [reply lib/src/screens/ionia/cards/ionia_gift_card_detail_page.dart]: done

File: lib/src/screens/ionia/cards/ionia_gift_card_detail_page.dart [hunk 2]
```diff
@@ -25,6 +26,8 @@ class IoniaGiftCardDetailPage extends BasePage {
 
   final IoniaGiftCardDetailsViewModel viewModel;
 
+
+
```
Comment (mkyq) [inline lib/src/screens/ionia/cards/ionia_gift_card_detail_page.dart:30]: Please remove empty lines

File: lib/src/screens/ionia/cards/ionia_gift_card_detail_page.dart [hunk 3]
```diff
@@ -64,6 +70,7 @@ class IoniaGiftCardDetailPage extends BasePage {
 
   @override
   Widget body(BuildContext context) {
+    viewModel.increaseBrightness ();
```
Comment (mkyq) [inline lib/src/screens/ionia/cards/ionia_gift_card_detail_page.dart:73]: here is type, a space between `increaseBrightness` and `()`

**Review by mkyq — CHANGES_REQUESTED** [review]

File: lib/view_model/ionia/ionia_gift_card_details_view_model.dart [hunk 4]
```diff
@@ -32,4 +34,9 @@ abstract class IoniaGiftCardDetailsViewModelBase with Store {
       redeemState = FailureState(e.toString());
     }
   }
+
+  void increaseBrightness () async {
```
Comment (mkyq) [inline lib/view_model/ionia/ionia_gift_card_details_view_model.dart:38]: Please remove a space between `increaseBrightness` and `()`

**Review by mkyq — CHANGES_REQUESTED** [review]

**Comment by Serhii-Borodenko** [comment]
done

**Review by mkyq — APPROVED** [review]

**Pull request merged**